webapp/channels/src/components/suggestion/suggestion_box/suggestion_box.jsx

```diff
@@ -564,7 +564,38 @@ export default class SuggestionBox extends React.PureComponent {
         return this.state.items.some((item) => !item.loading);
     };
 
+    confirmPretext = () => {
```

Comment:
Would it also make sense for `handleChange` to call `confirmPretext` as well?

Reply:
I get the feeling it should, but the reality is that the logic is slightly different. I didn't want to go down a more in depth refactoring of this component, because I usually get lost down that path :sweat_smile:

But I can look into everywhere the pretext is changed to do it the same way.

Reply:
Fair enough. I've felt like this whole component needs a rewrite for ages now, so I'm good with just fixing this specific issue for now instead of trying to solve everything that's wrong with it 😛